ega-charts/localega/templates/mina-inbox-deploy.yaml

```diff
@@ -55,12 +55,12 @@ spec:
         - name: BROKER_USERNAME
           value: {{ .Values.config.broker_username | quote }}
         - name: BROKER_VHOST
-          value: {{.Values.config.broker_vhost | quote }}
+          value: "/"
         - name: INBOX_PORT
           value: {{ .Values.inbox.port | quote }}
         - name: CEGA_ENDPOINT
         {{ if .Values.config.cega_users_host }}
-          value: {{ printf "%s%s%s" .Values.config.cega_users_host .Values.config.cega_users_endpoint "/%s?idType=username" | quote }}
+          value: {{ printf "%s%s" .Values.config.cega_users_host "/lega/v1/legas/users/%s?idType=username" | quote }}
```

Comment:
it is `v1` URL might change or not, but should we fix it ?

Reply:
No point in having it configurable unless there are more than one option

Reply:
fair point.